README.md

```diff
@@ -0,0 +1,47 @@
+# Simulating Calorie Reduction Values to Meet Obesity Reduction Targets in Scotland
```

Comment:
This is really clear / easy to follow. I wonder if you might want to include some of the figures in the readme and briefly talk around them? Nothing too detailed but could be nice to see what some of the outputs look like.

Reply:
Thanks - will do once findings finalised